test/lifecycle_test/component2.dart

```diff
@@ -151,10 +151,11 @@ ReactDartComponentFactoryProxy2 PropTypesTest = react.registerComponent(() => ne
 
 class PropTypesTestComponent extends react.Component2 {
   get propTypes => {
-        'intProp': (Map props, propName, componentName, location, propFullName) {
-          if (props[propName] is! int) {
+        'intProp': (Map props, info) {
+          dynamic propValue = props[info.propName];
+          if (propValue is! int) {
             return ArgumentError(
-                '$propName should be int. {"props": "$props", "propName": "$propName", "componentName": "$componentName", "location": "$location", "propFullName": "$propFullName"}');
+                '${info.propName} should be int. {"props": "$props", "propName": "${info.propName}", "componentName": "${info.componentName}", "location": "${info.location}", "propFullName": "${info.propFullName}"}');
           }
           return null;
```

Comment:
`return null` is no longer part of the boilerplate
```suggestion
```

Reply:
![nope](https://user-images.githubusercontent.com/1750797/68408189-25b04600-0142-11ea-8766-ad1a59e16154.gif)